docs/specs/tripgo.swagger.yaml

```diff
@@ -3281,6 +3281,9 @@ definitions:
       platform:
         type: string
         description: For transit segments (optional). Title for embarkation platform.
+      endPlatform:
+        type: string
+        description: For transit segments (optional). Title for arrival platform.
```

Comment:
To be consistent with the description above, this should say “Title for disembarkation platform”.

Reply:
@nighthawk you are right. I have just updated it.